server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/ImportEmlRoutes.java

```diff
@@ -83,33 +83,33 @@ public void define(Service service) {
         @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "EML file successfully imported"),
         @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "Invalid parameter")
     })
-    public HaltException importEmlFileToMailbox(Request request, Response response) throws MailboxException {
-
+    public HaltException importEmlFileToMailbox(Request request, Response response) throws Exception {
+        final byte[] fileSize = request.body().getBytes();
```

Comment:
so if I upload a 1GiB file, you will start by putting it in memory to tell me it's too big?
I'll probably break the server long before I receive the message.